crates/sncast/tests/data/accounts/accounts.json

```diff
@@ -47,19 +47,27 @@
             "public_key": "0x4fdaeb60e75399877bd5d14d3dac2e805f3c2a6732851a19db63fee0cfae00d",
             "address": "0x68d6e92b8e2310c550178b2fe484d5cb89dd6aa93ae804afd4d124e9d6c3cd3"
         },
-        "cairo1": {
+        "oz_cairo_1": {
             "private_key": "0x88ecc06581d81c76cef06d6f4f0c1b28",
             "public_key": "0x8cdbe26bc82084b04eccb3c6f8a76f12ad6c4015b3dc8ab90dc840d42cac29",
             "address": "0x691a61b12a7105b1372cc377f135213c11e8400a546f6b0e7ea0296046690ce",
             "legacy": false
         },
-        "cairo0": {
+        "oz_cairo_0": {
```

Comment:
I'd rather add a new field to accounts in here, something like "type", where it can be either one of oz, argent or braavos. The name could stay as well, but I realy think it may be a good idea to also unclude a type in there.

Reply:
It will be necessary if we add support for creating and deploying, but I wouldn't add it at this point, because it's currently not needed for anything.

Reply:
We want to start communicating with users that using argent is now officially possible. It may happen that we will not be ready with add/create commands in time for the next release. The users then will be forced to add accounts manually to the files, if they want to use such accounts. This will in turn make it very difficult for us to recognize the account type (we would have to assert this each and every time agains the chain, which is far from perfect). By adding this field now, we ensure that whatever happens we provide the users with a simple way to mark non-oz account, and do not have to do additional calls to chain in the future. It would be like "hey, this is an experimental way to use argent accounts, just mark them in the file if you really have to use them now and can't wait for add command". Does that make sense?

Reply:
that reminds me - we should probably also update changelog with this information, right?

Reply:
> We want to start communicating with users that using argent is now officially possible

I'm not convinced to be honest, I would rather prefer to say that it's possible to use accounts that perform validation using the standard Stark signer. We don't make any changes in the code logic here.

>  By adding this field now, we ensure that whatever happens we provide the users with a simple way to mark non-oz account, and do not have to do additional calls to chain in the future. It would be like "hey, this is an experimental way to use argent accounts, just mark them in the file if you really have to use them now and can't wait for add command"

It doesn't make much sense to me. Once we add logic for creating and deploying argent accounts in a basic form then we will inform about it. What's the rationale to do this earlier?

Reply:
> I'm not convinced to be honest, I would rather prefer to say that it's possible to use accounts that perform validation using the standard Stark signer. We don't make any changes in the code logic here.

if it's possibe, it means it's officially supported. 

> It doesn't make much sense to me. Once we add logic for creating and deploying argent accounts in a basic form then we will inform about it. What's the rationale to do this earlier?

People want to use them accounts now, and there is no reason to prevent them from it. We should inform that this is indeed possible, and they can import already existing accounts and just use them.

Reply:
As discussed offline, we will add the `type` field along with changes to the `account add` command in a separate PR. This will allow the addition of an argent account to the accounts file.